### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by benz0li**
@MaartenGr @dkapitan Please review.

### CODE REVIEW

**Comment by dkapitan** [comment]
thanks for this @benz0li! All looks good to me.

File: docs/faq.md [hunk 1]
```diff
@@ -313,21 +313,38 @@ typically do not contribute to the meaning of a document and should therefore be
...
+* Open VS Code, build the `Python base` or `Python scipy` container and start working  
+  ℹ️ Change `PYTHON_VERSION` to `3.11` in the respective `devcontainer.json` to work with the latest patch release of Python 3.11
+* Note that data is persisted in the container
+  * When using an unmodified `devcontainer.json`: Work in `/home/vscode`  
+    👉 This is the *home directory* of user `vscode`
+  * Python packages are installed to the home directory by default  
+    👉 This is due to env variable `PIP_USER=1`
+  * Note that the directory `/workspaces` is also persisted
 
+## **Speaking of Dev Containers: Do they support GPU acceleration?**
```
Comment (MaartenGr) [inline docs/faq.md:337]: Could you make this title a bit more specific and not dependent on the content of the previous question? The users will likely see a list of questions and the title "Speaking of Dev Containers" is dependent on the content of the previous title and not the title itself. 

Instead, you could also make it a subsection instead to showcase that it is indeed related to the previous section.

**Review by MaartenGr — COMMENTED** [review]
Thanks for the PR, highly appreciated! I made a small comment but other than that it looks great.

File: docs/faq.md (hunk 1 above)
Comment (benz0li) [reply docs/faq.md:337]: By subsection you mean

```
### **Speaking of Dev Containers: Do they support GPU acceleration?**
```

Correct?

**Comment by MaartenGr** [comment]
Thanks for the work on this, it is greatly appreciated! LGTM.

**Pull request merged**